src/server.ts

```diff
@@ -5,7 +5,7 @@ import { createMiddleware } from './middleware'
 export function createServer(...handlers: Array<HttpHandler>): express.Express {
   const app = express()
 
-  app.use(express.json())
+  app.use(express.raw({ type: '*/*' }))
```

Comment:
I’ve tested your code in one of my project and it works fine. However I quickly got a `413 Request Entity Too Large` since the default limit is only `'100kb'`. What do you think about raising the default and/or allowing it to be customizable?

```suggestion
  app.use(express.raw({ type: '*/*', limit: '20mb' }))
```